packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -404,3 +345,187 @@ describe("wrangler", () => {
     });
   });
 });
+
+/**
+ * This helper function will run a bunch of tests parameterized by the npm and yarn setup.
+ */
+function initAndInstall(
```

Comment:
It wasn't obvious that this function actually runs tests. Could you extract the tests into another function (say, `testInitCommand`, that accepts a `PackageManagerProtoType` arg), and have this function accept that function as an arg instead? So up above, you'd have, for example  - 



```js
function testInitCommand(PackageManagerPrototype){
  it("should create a package.json, and install packages, if none is found and user confirms", async () => {
        mockConfirm(
          {
            text: "No package.json found. Would you like to create one?",
            result: true,
          },
          {
            text: "Would you like to use typescript?",
            result: false,
          }
        );
       // ... and so on


initAndInstall(
      { npm: false, yarn: true, npmLockFile: true, yarnLockFile: true },
      YarnPackageManager.prototype,
      testInitCommand
)
```

Reply:
These tests and this function are now gone 💀